keps/sig-apps/3335-statefulset-slice/README.md

```diff
@@ -759,18 +790,32 @@ These goals will help you determine what you need to measure (SLIs) in the next
 question.
 -->
 
+The `statefulset_reconcile_delay` metric (time between StatefulSet reconciliation
+loops) should not significantly increase when using this feature.
```

Comment:
That's too hand-wavy for SLO.
I'm fine with saying that we didn't state any SLO - the SLI to monitor this feature is the metric you mentioned below.

Reply:
Removed the SLO. Will rely on the SLI for checking the correctness of the number of replicas.